src/XCCDF/result.c

```diff
@@ -34,8 +34,13 @@
 #ifndef WIN32_LEAN_AND_MEAN
 #define WIN32_LEAN_AND_MEAN
 #endif
+#ifdef __MINGW32__
+#include <winsock2.h>
+#include <iphlpapi.h>
+#else
 #include <WinSock2.h>
 #include <Iphlpapi.h>
```

Comment:
Windows is not case sensitive, let's use the right names here and avoid __MINGW32__.

It makes no sense to use the wrong names on Windows and the correct names in case we are on a case sensitive platform. Let's just use the correct names...